src/main/java/org/apache/sling/scriptingbundle/plugin/processor/Constants.java

```diff
@@ -53,6 +53,10 @@ private Constants() {}
     public static final String BND_SCRIPT_ENGINE_MAPPINGS = "scriptEngineMappings";
     public static final String BND_SEARCH_PATHS = "searchPaths";
 
+    public static final String VAULT_CONTEXT_XML = ".content.xml";
```

Comment:
There is a constant in FileVault

Reply:
Addressed in https://github.com/apache/sling-scriptingbundle-maven-plugin/pull/8/commits/24d6b83c3aa2517f6a8d05415f3eb8976d6cbe85.